source/common/router/router.cc

```diff
@@ -620,7 +625,7 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::HeaderMap& headers, bool e
   }
 
   return Http::FilterHeadersStatus::StopIteration;
-}
+} // namespace Router
```

Comment:
?

Reply:
ah that's my VSCode when I added extra `}` and saved file. 🤷‍♂